spec/factories/category_transactions.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :category_transaction do
+    category_id { rand(1000) }
```

Comment:
@sinpai you can use association like in example: `association :author, factory: :user, last_name: "Writely"`
Check [factory_bot docs](http://www.rubydoc.info/gems/factory_bot/file/GETTING_STARTED.md)